### ISSUE TITLE
OutOfMemory

### ISSUE THREAD

**Issue by hafizipudzi89** [winder/Universal-G-Code-Sender#1646]
**Describe the bug**
Ugs platform 2.0.8 will pop-up a OutOfMemory problem if open gcode file 1mb. But it can load and run the g code but will pause in the middle of the job, even worse, ugs will automatically idle state but the spindle still spinning..
. I thought it was fix in 2.0 version.

**Operating system (please complete the following information):**
 Windows 10,

**Comment by breiler** [comment]
I have used much larger files than that. Could you share your file?

**Comment by hafizipudzi89** [comment]
![Uploading 16273755485698157968310656457247.jpg…]()

**Comment by hafizipudzi89** [comment]
this is the file
[New Text Document (3).txt](https://github.com/winder/Universal-G-Code-Sender/files/6884148/New.Text.Document.3.txt)

**Comment by hafizipudzi89** [comment]
I just noticed that in about tab, ugs display my system not correctly, 
![IMG_20210727_170336](https://user-images.githubusercontent.com/45300557/127127733-d2054566-d74a-468d-9df3-c242b4e39d11.jpg)

**Comment by hafizipudzi89** [comment]
Any solution for this sir?

**Comment by hafizipudzi89** [comment]
> I have used much larger files than that. Could you share your file?

Sir? Any ideas for me to solve this problem?

**Comment by breiler** [comment]
The message you got is just a standard warning message built in to Netbeans (the platform UGS is built upon). I have successfully used files ~20Mb without problems. We have tweaked the configuration file for the next version of UGS so that this message is not shown.

The issue with the machine pausing in the middle of the job sounds like you have an unstable connection to the GRBL controller. When this happens I would like get the console log messages to see if the controller is being reset.
